pkg/kubelet/server/stats/handler.go

```diff
@@ -172,10 +173,9 @@ func writeResponse(response *restful.Response, stats interface{}) {
 // handleError serializes an error object into an HTTP response.
 // request is provided for logging.
 func handleError(response *restful.Response, request string, err error) {
-	switch err {
-	case kubecontainer.ErrContainerNotFound:
+	if errors.Is(err, kubecontainer.ErrContainerNotFound) {
 		response.WriteError(http.StatusNotFound, err)
-	default:
+	} else {
```

Comment:
Changing `switch` to `if/else` would encourage people to keep adding `else if`s for new cases in future. I'd suggest to keep the switch unless it causes linters to complain.

Reply:
You are right, thanks.